ykrt/src/compile/jitc_yk/arbbitint.rs

```diff
@@ -123,6 +123,11 @@ impl ArbBitInt {
         Some(self.val.sign_extend(self.bitw, 64) as i64)
     }
 
+    /// Sign extend the underlying value and, if it is representable as an `isize`, return it.
+    pub(crate) fn to_sign_ext_isize(&self) -> Option<isize> {
+        Some(self.val.sign_extend(self.bitw, 64) as isize)
```

Comment:
This isn't safe, because we have no guarantees about `isize`s size. We will have to think *very* carefully about what this function should do, and I don't immediately know if the function can even exist safely.

Reply:
The easiest (and perhaps best) thing we could do -- if it's possible! -- is to get rid of this function. Based on the code elsewhere (see a later comment) I'm not even sure if we actually need a "to isize" function yet.

If we do need it, we'll have to add checks (maybe using `static_assert`?) that `isize` and `type_of(self.val)` are the same size.